architecture/src/commonMain/kotlin/viewmodel/LifecycleViewModel.kt

```diff
@@ -23,10 +23,22 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.cancelChildren
 
+@Deprecated(
+    message = "ViewModel was renamed as the name didn't match it's function.",
+    replaceWith = ReplaceWith("LifecycleViewModel")
+)
+class ViewModel(allowFreezing: Boolean = false) : LifecycleViewModel(allowFreezing)
```

Comment:
Using type aliases caused a compilation error on android (duplicated class names). Changing it to subclasses fixed the issue